editor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axonivy/process-editor",
-  "version": "11.1.0-next.696+75006c0",
```

Comment:
looks strange

Reply:
Yes this should not stay there only in the look file